keycloak-js/keycloak-js-tests.ts

```diff
@@ -0,0 +1 @@
+import { Keycloak } from 'keycloak-js';
```

Comment:
You're right; sorry about that. I've fixed it (I hope) and updated the test file.